CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ IL Repack changelog
 * Put public the method retrieving repacked assembly names.
 * Fixed regression since 2.0.4 that prevented proper merging of resources in dependent assemblies.
  * Because of this, no BAML resources were added from dependencies into the merged result.
+* Add set Log level verbose in IlRepack constructor.
```

Comment:
You mean that sections for 2.0.7/2.0.8 are empty ? these versions were removed from nuget right after release (release process glitch)